start.html

```diff
@@ -178,145 +181,341 @@ <h3 class="accordion">
                          </p>
+                         <p class="info_text"  style="text-align:justify">
+                           <p class="c_snip">
+                           cd /path/to/some/directory/pbench/agent/ansible
+                           ansible-playbook -i ~/.config/Inventory/myhosts.inv --list-hosts pbench_agent_install.yml
+                           </p>
+                         </p>
+                         <p class="info_text"  style="text-align:justify">
+                           If you want to install on the local machine, the inventory file should include <span class="c_snip_inline">localhost</span> in
+                           the <span class="c_snip_inline">[servers]</span> section. Note that you need to be able to ssh to the localhost as root as well,
+                           so you might need to do <span class="c_snip_inline">ssh-copy-id root@localhost</span>.
+                         </p>
+                       </div>
+                     </div>
+                   </div>
+                   <!-- Subsection: Run the playbook -->
+		   <div class="accordionCard">
+		     <h4 class="accordion">
+		       <button class="accordionBtn pf-c-accordion__toggle" aria-expanded="false">
+			 <i class="arrowIcon fas fa-angle-right pf-c-accordion__toggle-icon" aria-hidden="true"></i>
+			 <span class="pf-c-accordion__toggle-text">Run the playbook</span>
+		       </button>
+		     </h4>
+		     <div class="explanation pf-c-accordion__expanded-content pf-m-fixed" hidden>
+		       <div class="pf-c-accordion__expanded-content-body">
 			 <p class="info_text"  style="text-align:justify">
-                           You should be ready to roll.
+                           You can install <span class="c_snip_inline">pbench-agent</span> by using the following command:
                          </p>
-		       </div>
-		     </div>
-		   </div>
+                         <p class="info_text"  style="text-align:justify">
+                           <p class="c_snip">
+                             ansible-playbook -i ~/.config/Inventory/myhosts.inv pbench_agent_install.yml
+                           </p>
+                         </p>
+                         <p class="info_text"  style="text-align:justify">
+                           That will install the appropriate version
+                           of <span class="c_snip_inline">pbench.repo</span>
+                           into the
+                           <span class="c_snip_inline">/etc/yum.repos.d</span> directory of each host (you don't have to worry
+                           about what distro the host is running: the repo file is tailored
+                           accordingly); it will then run either ~yum~ or ~dnf~ to install or
+                           update the ~pbench-agent~ package; and it will install the correct ssh
+                           key file and the correct pbench-agent config files for the production
```

Comment:
`s/for the production environment/for your environment/`?

Reply:
Fixed.